go.mod

```diff
@@ -16,7 +18,7 @@ require (
 	github.com/dell/dell-csi-extensions/volumeGroupSnapshot v1.6.0
 	github.com/dell/gocsi v1.11.0
 	github.com/dell/gofsutil v1.16.1
-	github.com/dell/goscaleio v1.15.0
+	github.com/dell/goscaleio v1.16.1-0.20240923113203-f072f0cb0a88
```

Comment:
**question:** Should `goscaleio` be tagged so that we use a specific tagged version?

Reply:
No, it should point to a release-version. However, nothing is released yet and this would need to be updated. I will plan on following up with this in the next sprint and will address it in a separate PR.